crates/goose/tests/providers.rs

```diff
@@ -233,8 +233,8 @@ impl ProviderTester {
         dbg!(&result);
         println!("===================");
 
-        // Ollama and OpenRouter truncate by default even when the context window is exceeded
-        if self.name.to_lowercase() == "ollama" || self.name.to_lowercase() == "openrouter" {
```

Comment:
@salman1993 I figured out that this test assumption is actually wrong. It only didn't return an error because the error wasn't being captured.